scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -538,19 +538,22 @@
                                                  (select-keys (keywordize-keys (:resources v))
                                                               [:cpus :mem :gpus])]))
                                          (into {}))]
-          (when (and (seq params)
-                     (> utilization (:min-utilization-threshold params)))
+          (if (and (seq params)
+                   (> utilization (:min-utilization-threshold params)))
             (let [{normal-pending-jobs :normal gpu-pending-jobs :gpu} @pending-jobs-atom]
               (rebalance! conn driver offer-cache normal-pending-jobs host->spare-resources
                           rebalancer-reservation-atom
-                          (assoc params :compute-pending-job-dru compute-pending-normal-job-dru
-                                        :category :normal))
+                          (assoc params :category :normal
+                                        :compute-pending-job-dru compute-pending-normal-job-dru))
               (rebalance! conn driver offer-cache gpu-pending-jobs host->spare-resources
                           rebalancer-reservation-atom
-                          (assoc params :compute-pending-job-dru compute-pending-gpu-job-dru
-                                        :category :gpu))))))
-        {:error-handler (fn [ex] (log/error ex "Rebalance failed"))})
-      #(async/close! trigger-chan)))
+                          (assoc params :category :gpu
+                                        :compute-pending-job-dru compute-pending-gpu-job-dru)))
+            (log/info "Rebalance iteration is a no-op"
```

Comment:
Maybe "Skipping rebalancing due to low cluster utilization"?

Reply:
That may not be true as `min-utilization-threshold` may be nil.

Reply:
Should we just disable the rebalancer in that case? I like @dposada 's message better.

Reply:
We can't disable the rebalancer as the data might be written to the DB after Cook has been running for a while. I'll change the messaging.

Reply:
Changed the log message.